backend/src/main/java/se/chalmers/dat265/group1/storage/repository/GenericRepository.java

```diff
@@ -84,15 +85,15 @@ public T getObject(int id) {
         System.out.println("Running query: "+query);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)